app/pulp/app/response.py

```diff
@@ -0,0 +1,32 @@
+from gettext import gettext as _
+
+from rest_framework.response import Response
+
+
+class OperationPostponedResponse(Response):
+    """
+    An HTTP response class for returning 202 and a list of spawned tasks.
+
+    This reponse object should be used by views that dispatch asynchronous tasks. Most common use
+    case is for sync and publish operations. When the task_group_id is specified, the task_results
```

Comment:
Having this one object provide both types when one is optional in most cases may be better as two separate objects. It could be called GroupOperationPostponedResponse and be a subclass or peer object of OperationPostponedResponse. What do you think?